src/hooks/index.js

```diff
@@ -760,3 +760,8 @@ exports.populateCurrentSchool = async (context) => {
 	}
 	throw new BadRequest('Authentication is required.');
 };
+
+exports.addCollation = (context) => {
+	context.params.collation = { locale: 'de', caseLevel: true };
+	return context;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5320952)